content/javascript/concepts/window-clearInterval/window-clearInterval.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'clearInterval()'
+Description: 'Cancels a timer previously stablished by setInterval().'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**`clearInterval()`** is a window method used to shut down the loop-calling process of a `setInterval()` execution when it must be stopped or aborted.
+
+## Syntax
+
+`clearInterval()` receives a parameter called `intervalId` which is returned from the previously created `setInterval()` timer.
+
+The below is an example of how `clearInterval()` is composed:
+
+```pseudo
+clearInterval(intervalID)
+```
+
+## Example
+
+The following sample shows the usage of `clearInterval()` by canceling a previously added `setInterval()` to a function.
```

Comment:
The example still never stops the timer. Edit it so it actually uses the function you created.